kv/store.go

```diff
@@ -111,7 +131,10 @@ func (store Store) UpdateConnection(conn *connection.Connection) (*connection.Co
 func (store Store) DeleteConnection(space string, id connection.ID) error {
 	deleteConnection := etcd.OpDelete(string(id))
 	deleteJobs := etcd.OpDelete(fmt.Sprintf("%s/%s", id, jobsDir), etcd.WithPrefix())
-	resp, err := store.client.Txn(context.TODO()).Then(deleteConnection, deleteJobs).Commit()
+	deleteWorkers := etcd.OpDelete(fmt.Sprintf("%s/%s", workerDir, id), etcd.WithPrefix())
+
+	fmt.Printf("DEBUG -- delConn: %s, delJob: %s, delWork: %s\n", deleteConnection.KeyBytes(), deleteJobs.KeyBytes(), deleteWorkers.KeyBytes())
```

Comment:
The reason why it's not working is that the `store.client` is already prefixed with `connections/` (`main.go` line 93). So you are not able to access `worker/` prefix if all queries run in `connections/` context. As I mentioned during the meeting. The connection prefix from `main.go` should be moved to `kv` package. It's not used anywhere else.

Reply:
Hmm, so this all gets prepended with `connections`?

```
DEBUG -- delConn: 1BOjkeUCojGB2Li3ezxBfA3UFJy, delJob: 1BOjkeUCojGB2Li3ezxBfA3UFJy/jobs/, delWork: workers/1BOjkeUCojGB2Li3ezxBfA3UFJy
```